libs/standard-tests/langchain_tests/integration_tests/chat_models.py

```diff
@@ -1323,8 +1331,8 @@ def has_tool_calling(self) -> bool:
             .. code-block:: python
 
                 @pytest.mark.xfail(reason=("Not implemented."))
-                def test_tool_message_histories_string_content(self, model: BaseChatModel) -> None:
-                    super().test_tool_message_histories_string_content(model)
+                def test_tool_message_histories_string_content(self, *args: Any) -> None:
```

Comment:
smart - should we pass **kwargs too? Just in case we add kwargs for some reason in future